src/cordova/build.action.mjs

```diff
@@ -33,35 +35,46 @@ export async function main(...parameters) {
   await runAction('cordova/setup', ...parameters);
 
   if (buildMode === 'debug') {
-    console.warn(`WARNING: building "${cordovaPlatform}" in [DEBUG] mode. Do not publish this build!!`);
+    console.warn(`WARNING: building "${outlinePlatform}" in [DEBUG] mode. Do not publish this build!!`);
   }
 
-  if (outlinePlatform === 'macos' || outlinePlatform === 'ios') {
-    const WORKSPACE_PATH = `${process.env.ROOT_DIR}/src/cordova/apple/${outlinePlatform}.xcworkspace`;
+  if (cordovaPlatform === 'osx' || cordovaPlatform === 'ios') {
+    const xcodebuildFlags = {
+      workspace: path.join(getRootDir(), 'src', 'cordova', 'apple', `${outlinePlatform}.xcworkspace`),
+      scheme: 'Outline',
+    };
+
     // TODO(fortuna): Specify the -destination parameter for build. Do we need it for archive?
+    await runXcodebuild.clean(xcodebuildFlags);
```

Comment:
We were not cleaning for dev before. Is the change intended?

Reply:
I'd point our that the lack of arbitrary args in the runXcodebuild command prevents us from cleaning in the same build command like we were doing before.